.codespell.exclude

```diff
@@ -53,3 +55,6 @@ planet "Sies Upi"
 		"Brocken"
 			`	"What part of non-negotiable did you not understand? You're pushing your luck, Captain <last>. I didn't expect you to be such a petty money pincher."`
 		"Shepard"
+# Following two lines to be deleted once unicode support is added
+	#`	"Eu preferiria se tu não fosse tão negativo, camarada. Tu sabe tão bem quanto eu que é só uma questão de esperar as coisas entrarem em colapso. Porra, tem uma guerra civil arriscando começar a qualquer momento, se segura por só mais um pouquinho! Inclusive, eu escutei a pouco que – "`
```

Comment:
We can have the exclude with the accent marks when they're actually in use.
```suggestion
```

Reply:
A big part of it _is_ that's how the language is supposed to be displayed, just limited by the lack of technology. Especially with how the Unicode PR got a big push and still is a long ways off. If the file absolutely has to be scrubbed clean, I'd prefer if these lines be kept in until right before it's merged, just so they're as easily accessible as possible.